components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -912,7 +909,7 @@ export class GitpodServerImpl implements GitpodServerWithTracing, Disposable {
         const user = await this.checkAndBlockUser("startWorkspace", undefined, { workspaceId });
 
         // (gpl) We keep this check here for backwards compatibility, it should be superfluous in the future
-        const workspace = await this.workspaceService.getWorkspace(user.id, workspaceId);
+        const { workspace } = await this.workspaceService.getWorkspace(user.id, workspaceId);
```

Comment:
we are fetching the instance below again. Is that still needed or can we use the one returned by this call?

Reply:
do we even need `getCurrentInstance` at all anymore?